packages/api/test/utils/fixtures.js

```diff
@@ -68,7 +67,7 @@ const fixtures = {
       transaction = await fixtures.transaction(knex, {
         block_hash,
         owner: identifier,
-        type: StateTransitionEnum.IDENTITY_CREATE
+        type: Constants.StateTransitionEnum.IDENTITY_CREATE
```

Comment:
`const {StateTransitionEnum} from ./constants` does it work like that?

Reply:
No